parkings/tests/test_models.py

```diff
@@ -19,3 +21,28 @@ def test_parking_instance_creation():
         time_start=now(),
         zone=3,
     )
+
+
+@pytest.mark.django_db
+@override_settings(TIME_ZONE='Europe/Helsinki')
+def test_parking_str(parking_factory):
+    parking = parking_factory(
+        time_start=datetime.datetime(2014, 1, 1, 6, 0, 0, tzinfo=utc),
+        time_end=datetime.datetime(2016, 1, 1, 7, 0, 0, tzinfo=utc),
+        registration_number='ABC-123',
+    )
+    assert all(parking.__str__().count(val) == 1 for val in ('2014', '2016', '8', '9', 'ABC-123'))
```

Comment:
minor, but I would suggest to avoid using dunder methods when possible. Maybe `str(parking).count(val)`